tigerpath/majors_and_certificates/majors/ARC_2018.json

```
@@ -0,0 +1,194 @@
+              "course_list": [
+                "AMS 322/URB 322/ARC 326/AAS 320",
+                "ARC 302/ART 347",
+                "ARC 308/ART 328",
+                "ARC 322",
+                "ARC 489",
+                "ARC 492/URB 492/ENV 492",
+                "ART 102/ARC 209",
+                "ART 341/ARC 341",
+                "ART 445/ARC 445",
+                "ART 466/ARC 466/SPA 466",
+                "ECS 343/POL 406/ARC 343/ART 352",
+                "ECS 376/ARC 376",
+                "HIS 464/ARC 464/ENV 464/URB 464"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "History and Theory of Urbanism and Landscape",
+          "max_counted": 1,
+          "min_needed": 2,
+          "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>\n\nTwo courses in History and Theory of Urbanism and Landscape.",
+          "course_list": [
+            "ARC 205/URB 205/LAS 225/ENV 205",
+            "ARC 303",
+            "ARC 312",
+            "ARC 321",
+            "ARC 396/SAS 396/URB 396",
+            "ARC 449",
+            "ARC 492/URB 492/ENV 492",
+            "ART 250",
+            "DAN 310/ARC 380/THR 323/URB 310",
+            "ENV 382",
+            "ENV 424/ARC 424/ART 492",
+            "URB 201/WWS 201/SOC 203/ARC 207",
+            "URB 300/ARC 300/HUM 300/SLA 300",
+            "URB 385/SOC 385/HUM 385/ARC 385",
+            "URB 401/ARC 401"
+          ]
+        },
+        {
+          "name": "Art and Archaeology",
+          "max_counted": 1,
+          "min_needed": 2,
+          "explanation": "At least two upper level courses to be taken in the Department of Art and Archaeology.",
+          "course_list": [
+            "ART 3*",
+            "ART 4*",
+            "ART 5*"
+          ]
+        },
+        {
+          "name": "Building Technology",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "The Technology distribution requires two courses, one of which is ARC 311: Building Science and Technology–Building Systems.",
+          "req_list": [
+            {
+              "name": "Required Courses",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "course_list": [
+                "ARC 311"
+              ]
+            },
+            {
+              "name": "Electives",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>",
+              "course_list": [
+                "ARC 374",
+                "ARC 406/ENV 406",
+                "CEE 261/ARC 262/EGR 262/URB 262/ART 262",

```

Typo here, I believe
```suggestion
                "CEE 262/ARC 262/EGR 262/URB 262/ART 262",
```
_Edit_: In fact, on second glance, it has two versions, A and B, so it should be either
```suggestion
                "CEE 262A/ARC 262A/EGR 262A/URB 262A/ART 262",
                "CEE 262B/ARC 262B/EGR 262B/URB 262B",
```
or more succinctly
```suggestion
                "CEE 262*/ARC 262*/EGR 262*/URB 262*/ART 262",
```